src/components/ToolMenu/Draw/index.tsx

```diff
@@ -345,12 +345,25 @@ export const Draw: React.FC<DrawProps> = ({
           </span>
         </DeleteButton>
       ) : <></>}
+      {showDeleteFeatures ? (
+        <DeleteAllButton
+          name = "draw-delte-all"
```

Comment:
```suggestion
          name="draw-delete-all"
```

Reply:
done in: [36df83f](https://github.com/terrestris/shogun-gis-client/pull/1476/commits/36df83fa949184301bcbc5a0c8350cf936547cd6)